src/components/timeline/form/TimelineEditorPanel.svelte

```diff
@@ -978,6 +960,141 @@
             </Input>
           </form>
         </fieldset>
+      {:else if rowHasExternalEventLayer}
+        <fieldset class="editor-section">
```

Comment:
I'd like to see this form merge in some way with the Activity Options form given the amount of duplication here. Don't see any reason why these two couldn't be the same thing and if there are options that only matter to one of these we could hide it with a prop?

Reply:
Great and agreed, I think we need to think more about how to handle (even just at a minimum) the case of events and activities in the same row wrt layer options. Might want to chat with @lklyne about this.

Reply:
This has been addressed. There is now a subheader indication what options are of what type:
<img width="289" alt="image" src="https://github.com/user-attachments/assets/92fea14d-429d-408c-9975-81405a91ec25">